apps/ledger-live-desktop/src/renderer/families/elrond/components/Modals/Delegate/Body.jsx

```diff
@@ -125,7 +125,6 @@ const Body = (props: Props) => {
       account,
       parentAccount: undefined,
       transaction: bridge.updateTransaction(transaction, {
-        mode: "delegate",
```

Comment:
Any reason why removing it from here?

Reply:
As far as I'm aware, setting the mode will trigger a validation of the transaction before the user has the chance to input an amount, thus erroring out into the `Minimum delegation amount should be at least 1 EGLD`. Thus, the key now will only be added once the amount key is also updated.